my-project/src/store/store.js

```diff
@@ -1,8 +1,45 @@
-import { createStore } from "vuex";
+import { createStore } from "vuex"
+import { year } from "../definition.js"
 
 export default createStore({
-  state: {},
-  mutations: {},
+  state() {
+    return {
+      gender: null,
+      year: year[0],
+      month: 1,
+      day: 1,
+      insurance: null,
+      medical: null,
+      hospitalization: null,
+      consultation: null
+    }
+  },
+  mutations: {
+    setGender: function (state, gender) {
```

Comment:
以下のように省略して記述することが多いです。
今回は修正はしなくて良いですが、次回から省略して記述すると、より現場に近いです。
```suggestion
    setGender(state, gender) {
```